### PR TITLE
fix app-breaking bug

### PR DESCRIPTION
**Pull request by netherwaves**

### CODE REVIEW

File: autonomx/layout/LatticeView.qml [hunk 1]
```diff
@@ -27,6 +27,7 @@ Item {
     // backend read
     onGeneratorIndexChanged: {
         panManager.setFromLatticeView(generatorModel.at(generatorIndex));
+        allowSnapW = allowSnapH = false
```
Comment (aalex) [inline autonomx/layout/LatticeView.qml]: I would put that statement in two separate lines and add a semicolon after each.

allowSnapW = false;
allowSnapH = false;

**Review by aalex — CHANGES_REQUESTED** [review]
I would put that statement in two separate lines and add a semicolon after each.

allowSnapW = false;
allowSnapH = false;

**Review by jvlyndark — APPROVED** [review]
LGTM

**Review by aalex — APPROVED** [review]
LGTM

**Pull request merged**